modules/programs/aerc.nix

```diff
@@ -1,5 +1,4 @@
 { config, lib, pkgs, ... }:
-
```

Comment:
These seem like unrelated changes. I would prefer to have a targeted commit with just the necessary changes. These cosmetic changes will just make `git blame` more difficult. Would you mind amending the commit?

Reply:
Although, I do agree that the unnecessary parentheses are a bit annoying 🙂

Reply:
Definitely! These changes stemmed from using the Alejandra Nix formatter which is setup in my editor, so I didn't notice before committing :sweat_smile: Rolling back now!